src/kawaz/core/personas/migrations/0005_auto_20161002_2154.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.2 on 2016-10-02 12:54
```

Comment:
This migration was generated automatically.
So I think I don't need to comment reasons to migrate.

Just it is changed behavior on Django 1.10